packages/docs/docs/compliance/clia-cap.md

```diff
@@ -52,3 +53,29 @@ Review and refer to the following items in preparation for your inspection.
   - Refer to documentation on [availability](/security#availability)
+- GEN.20316 QMS Indicators of Quality - Phase II
+
+  - Patient/Specimen Identification: Percent of patient wristbands with errors (ie, mislabels), percent of specimens with patient labeling errors (ie, mislabels), or percent of results with identification errors
+    - [ ] The main field for recording issues is [Specimen.condition](https://app.medplum.com/Specimen?_count=20&_fields=id,_lastUpdated,condition&_offset=0&_sort=-_lastUpdated) should include codes from the [`SpecimenRejectionReason` code system](https://terminology.hl7.org/CodeSystem-v2-0490.html).
+    - [ ] In this case the correct code for rejected specimen is `RI - Identification Problem`
+  - Test Order Accuracy: Percent of test orders correctly entered into a laboratory computer
+    - Though manual entry of orders is expected to be rare, the [QuestionnaireResponse](https://app.medplum.com/QuestionnaireResponse) resource is the source of truth for manual order entries.
+  - Specimen Acceptability: Percent of specimens received that are suitable for testing
+    - [ ] All [Specimens](https://app.medplum.com/Specimen)
+    - [ ] [Unsatisfactory Specimens](https://app.medplum.com/Specimen?_count=20&_fields=id,_lastUpdated,status&_offset=0&_sort=-_lastUpdated&status=unsatisfactory)
+  - Test Turnaround Time: Collection-to-reporting turnaround time or receipt-in- laboratory-to-reporting turnaround time of tests ordered. This may include orders of a “stat” priority (eg, emergency department or intensive care unit specimens), or routine priority, to include the percent of specimens with turnaround time that falls within an established limit (eg, the time that represents the 90th or 95th percentile of turnaround times or less than 30 minutes).
+    - [ ] Collection-to-reporting turnaround: [Specimen.collection.collectedDateTime](https://app.medplum.com/Specimen?_count=20&_fields=id,_lastUpdated,condition,collection&_offset=0&_sort=-_lastUpdated) to [DiagnosticReport.issued](https://app.medplum.com/DiagnosticReport?_count=20&_fields=id,_lastUpdated,subject,code,status,specimen,issued&_offset=0&_sort=-_lastUpdated)
+    - [ ] Receipt-in-laboratory-to-reporting turnaround: [Specimen.receivedTime](https://app.medplum.com/Specimen?_count=20&_fields=id,_lastUpdated,receivedTime&_offset=0&_sort=-_lastUpdated) to [DiagnosticReport.issued](https://app.medplum.com/DiagnosticReport?_count=20&_fields=id,_lastUpdated,subject,code,status,specimen,issued&_offset=0&_sort=-_lastUpdated)
+  - Critical Result Reporting: Percent of critical results with written record that results have been reported to caregivers; percent of critical results for which the primary clinician cannot be contacted in a reasonable period of time
+    - [ ] Patient contact are often implemented as [Communication](https://app.medplum.com/Communication?_count=20&_fields=id,_lastUpdated,status,subject,sender,payload&_offset=0&_sort=-_lastUpdated)
```

Comment:
In scenarios where users are creating `Communication` resources for different purposes (e.g. patient SMS, EHR Chat), we'll need to define some ontology to filter these out using `Communication.category`